frontend/src/metabase/actions/components/ActionViz/ActionOptions.tsx

```diff
@@ -0,0 +1,239 @@
+import React, { useCallback, useMemo } from "react";
+import { t } from "ttag";
+import { connect } from "react-redux";
+import _ from "underscore";
+
+import { useToggle } from "metabase/hooks/use-toggle";
+
+import Icon from "metabase/components/Icon";
+import Select from "metabase/core/components/Select";
+import Button from "metabase/core/components/Button";
+
+import TippyPopoverWithTrigger from "metabase/components/PopoverWithTrigger/TippyPopoverWithTrigger";
+import ActionCreator from "metabase/actions/containers/ActionCreator";
```

Comment:
Not sure if that's a big deal, but it feels off that `actions/components` import stuff from `actions/containers`